ert3/workspace/__init__.py

```diff
@@ -3,6 +3,10 @@
 from ert3.workspace._workspace import get_experiment_names
 from ert3.workspace._workspace import experiment_has_run
 from ert3.workspace._workspace import assert_experiment_exists
+from ert3.workspace._workspace import load_ensemble_config
+from ert3.workspace._workspace import load_stages_config
+from ert3.workspace._workspace import load_experiment_config
+from ert3.workspace._workspace import load_parameters_config
 
 EXPERIMENTS_BASE = "experiments"
```

Comment:
> Out of curiosity, can this now be a local value in module? 🤔

Yes, it can. It is only used in the workspace code file and in tests. I suppose it is better not to expose it, and let the tests just access it anyway.